modules/nf-core/merquryfk/merquryfk/meta.yml

```
@@ -106,7 +90,33 @@ output:
       type: file
       description: "error and qv of each assembly as a whole"
       pattern: "*.qv"
+  - phased_block_bed:
+      type: file
+      description: Assembly kmer positions seperated by block in bed format
+      pattern: "*.phased.block.bed"
+  - phased_block_stats:
+      type: file
+      description: phased assembly statistics file
+      pattern: "*.phased.block.stats"
+  - continuity_N:
+      type: file
+      description: "Stacked assembly N continuity plot in PNG or PDF format"
+      pattern: "*.continuity.N.*"
+  - block_N:
+      type: file
+      description: "Stacked assembly N continuity by block plot in PNG or PDF format"
+      pattern: "*.block.N.*"
+  - block_blob:
+      type: file
+      description: "Stacked assembly block plot in PNG or PDF format"
+      pattern: "*.block.blob.*"
+  - hapmers_blob:
+      type: file
+      description: "Stacked assembly hapmers block plot in PNG or PDF format"
+      pattern: "*.hapmers.blob.*"
 authors:
   - "@mahesh-panchal"
+  - "trio option added by @yumisims"
```

```suggestion
  - "@yumisims"
```
I think the extra text would just cause problems for anything automated that's using this.